components/camel-sip/src/main/java/org/apache/camel/component/sip/SipConfiguration.java

```diff
@@ -389,11 +699,24 @@ private void createContactHeader() throws ParseException {
         contactHeader = headerFactory.createContactHeader(contactAddress);
     }
 
+    /**
+     * Creates the ExpiresHeader based on the msgExpiration instance variable
+     *
+     * @throws ParseException
+     * @throws InvalidArgumentException
+     */
     private void createExpiresHeader() throws ParseException, InvalidArgumentException {
         expiresHeader = getHeaderFactory().createExpiresHeader(getMsgExpiration());        
     }
-    
-    Properties createInitialProperties() {
+
+    /**
+     * creates a properties object holding information about the stack name, the limit on message size,
+     * whether connections should be cashed, if messages can be send via proxies and optionally where the logs
+     * are located
+     *
+     * @return a properties object holding the information stated above
+     */
+    public Properties createInitialProperties() {
```

Comment:
@nikvaessen does this absolutely need to become public?  @davsclaus is there any other pattern you would prefer to see used instead of making this method public?